src/main/scala/org/zalando/spearheads/innkeeper/routes/RequestParameters.scala

```diff
@@ -0,0 +1,18 @@
+package org.zalando.spearheads.innkeeper.routes
+
+import java.time.LocalDateTime
+import java.time.format.DateTimeFormatter
+
+import scala.util.Try
+
+/**
+ * @author Alexey Venderov
+ */
+object RequestParameters {
+
+  val urlDateTimeFormatter = DateTimeFormatter.ISO_LOCAL_DATE_TIME
+
+  def dateTimeParameter(dateTimeParameter: String): Option[LocalDateTime] =
```

Comment:
OK. Can be done later as an improvement. Currently I only extracted the code that can be reused into a separate class.